widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -208,6 +115,7 @@ private static void setupQueueIds(@Nullable List<String> queueIds) {
      *
      * <p>This method is no-op for other non-Glia triggered results.</p>
      */
+    //TODO check the possibilities to remove this method
```

Comment:
Is this going to be done in this PR or should we create a separate jira ticket for this?